data/categories.txt

```diff
@@ -904,7 +904,7 @@ es:3Panadería
 eu:3Okindegia
 sv:3Bageri
 th:3เบเกอรี่
-tr:3Fırın
+tr:2Fırın
```

Comment:
The same. Why 2?

Reply:
I thought it would better
I can change if it's problem